test/hooks.js

```diff
@@ -72,7 +72,7 @@ tape('[promises] test level-hookdown', async function (t) {
 })
 
 tape('test level-hookdown series', function (t) {
-  var level = MemDB({ valueEncoding: 'json' })
+  var level = mem(null, { valueEncoding: 'json' })
```

Comment:
This difference between `memdb` and `level-mem` will be fixed by https://github.com/Level/packager/pull/95